packages/cipherstash-proxy-integration/src/map_params.rs

```diff
@@ -182,6 +182,35 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn map_timestamp() {
+        trace();
+
+        let client = connect_with_tls(PROXY).await;
+
+        let id = id();
+        let encrypted_timestamp = "2000-10-01T17:30:00Z".parse::<DateTime<Utc>>().unwrap();
```

Comment:
Would it be a good idea to add another test case set in a different time zone than UTC?